pkgs/applications/networking/instant-messengers/tangram/default.nix

```diff
@@ -15,7 +31,14 @@ stdenv.mkDerivation rec {
     fetchSubmodules = true;
   };
 
-  buildInputs = [ gdk-pixbuf gjs glib gsettings-desktop-schemas gtk3 webkitgtk ];
```

Comment:
I don't think anything got changed here. It would be probably better to revert this.